persistence-cassandra-3.11/pom.xml

```diff
@@ -46,6 +46,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.datastax.cassandra</groupId>
+      <artifactId>cassandra-driver-core</artifactId>
+      <version>3.0.1</version>
```

Comment:
Just curious, why is this an older version?

Reply:
It matches the version in `cassandra-all`'s POM. Unfortunately there's no way to infer it, we'll have to maintain the two dependencies in sync manually. I'll add a comment in the code.